cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -306,7 +306,8 @@ func findMatchingCR(cmClient cmclient.Interface, ctx context.Context, crt *cmapi
 	if crt.Status.Revision != nil {
 		nextRevision = *crt.Status.Revision + 1
 	}
-	for _, req := range reqs.Items {
+	for i := range reqs.Items {
+		req := reqs.Items[i]
 		if predicate.CertificateRequestRevision(nextRevision)(&req) &&
 			predicate.ResourceOwnedBy(crt)(&req) {
 			possibleMatches = append(possibleMatches, req.DeepCopy())
```

Comment:
I don't think the linter is complaining about it being used after DeepCopy() is called.
I think it is complaining because it is used in `predicate.CertificateRequestRevision(nextRevision)(&req)` and `predicate.ResourceOwnedBy(crt)(&req)`.